src/components/timeline/LayerActivity.svelte

```diff
@@ -86,113 +100,46 @@
 
   onDestroy(() => removeKeyDownEvent());
 
-  /**
-   * Recursively converts an Activity to an ActivityPoint.
-   * Sorts child activity points in start time ascending order.
-   */
-  function activityToPoint(activity: Activity): ActivityPoint {
-    const children = activity?.childUniqueIds
-      ? [...activity.childUniqueIds]
-          .sort((aId: ActivityUniqueId, bId: ActivityUniqueId): number => {
-            const a = $activitiesMap[aId];
-            const b = $activitiesMap[bId];
-
-            if (a && b) {
-              return sortActivities(a, b);
-            }
-
-            return 0;
-          })
-          .reduce((childActivityPoints, childUniqueId: ActivityUniqueId) => {
-            const childActivity = $activitiesMap[childUniqueId];
-
-            if (childActivity) {
-              const childActivityPoint = activityToPoint(childActivity);
-              childActivityPoints.push(childActivityPoint);
-            }
-
-            return childActivityPoints;
-          }, [])
-      : [];
-
-    let activityLabelText = activity.name;
-
-    if (activity.parent_id !== null) {
-      activityLabelText = activity.type;
-    }
-
-    if (activity.unfinished) {
```

Comment:
Do we still want to render this unfinished state? Not seeing it in the new code. That said I could just work that into my PR?

Reply:
Good catch! Yes we still need it

Reply:
One other thing is that we used to show "(Unfinished)" for the point name. Still want that too I assume?

Reply:
So this branch actually does not render the parent span and still rolls it up in the directive, and since the directive can't technically be "unfinished", it feels weird to add it there since we are changing it anyway. Can we add it on your branch to the parent span instead? Essentially it's just checking if the span duration is `null`. Is there anything specifically you need on this branch to support it on your branch?

Reply:
And I guess we'd need to cover all the spans for a directive to find out if the directive is unfinished, assuming we still want to show the unfinished label for the directive?

Reply:
No I'm saying we just show it for the parent span instead of the directive, since the span is the thing that is actually simulated and unfinished.

Reply:
Whoops sorry, did not refresh github to see your other comment before I added my second one. Got it, can add that to the parent span only.